src/main/kotlin/br/unipar/plano/interfaces/rest/solicitacaoprocedimento/SolicitacaoProcedimentoResource.kt

```diff
@@ -0,0 +1,56 @@
+package br.unipar.plano.interfaces.rest.solicitacaoprocedimento
+
+import br.unipar.plano.domain.solicitacaoprocedimento.model.IdSolicitacaoProcedimento
+import br.unipar.plano.domain.solicitacaoprocedimento.service.SolicitacaoProcedimentoService
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder
+import java.util.*
+import javax.validation.Valid
+
+@RestController
+@RequestMapping("/solicitacao-liberacao-procedimento")
+class SolicitacaoProcedimentoResource(val solicitacaoProcedimentoService: SolicitacaoProcedimentoService) {
+
+    @PostMapping("/criar")
```

Comment:
Nunca informe verbos na url, o `Post` já indica que você está criando algo novo.

```suggestion
    @PostMapping
```